src/components/menu/menu.stories.tsx

```diff
@@ -14,7 +14,7 @@ export const Default: Story = {
   render: () => (
     <div className="w-52">
       <Menu>
-        <Menu.Button>Open Menu</Menu.Button>
+        <Menu.Button className="IShouldBeOptional">Open Menu</Menu.Button>
```

Comment:
This is a fix for a type issue. 
The component `Menu.Button` has the `className` prop as non-optional, so the story requires the thing to be there. I added it to the story to avoid modifying "prod" code, but if it's safe, I can also modify the code and remove it from here.